policyengine_us/parameters/gov/states/ca/tax/income/credits/foster_youth/phase_out/increment.yaml

```
@@ -0,0 +1,15 @@
+description: California reduces the foster youth tax credit for each of these increments of adjusted gross income exceeding the threshold.
```

Please note the material policy difference here
```suggestion
description: California reduces the foster youth tax credit for each of these increments of earned income exceeding the threshold.
```